pkg/middleware/lsds/lsds.go

```diff
@@ -1,15 +1,19 @@
-package middleware
+package lsds
 
-import "github.com/tass-io/scheduler/pkg/span"
+import (
+	"github.com/tass-io/scheduler/pkg/middleware"
+	"github.com/tass-io/scheduler/pkg/span"
+)
 
 const (
-	LSDSMiddlewareSource Source = "LSDS"
+	LSDSMiddlewareSource middleware.Source = "LSDS"
 )
 
 var lsdsmiddle *LSDSMiddleware
 
 func init() {
 	lsdsmiddle = NewLSDSMiddleware()
```

Comment:
why public this function?

Reply:
I think you should unify your style of initializing a single instance

for example, your initialization style in `pkg/event/schedule/schedule_handler.go` is different

Reply:
ok, I will unify my style.